app/code/Magento/Tax/view/frontend/web/js/view/checkout/summary/tax.js

```diff
@@ -99,6 +103,33 @@ define([
             return this.getFormattedPrice(amount);
         },
 
+        /**
+         * @param {*} parent
+         * @param {*} percentage
+         * @return {*|String}
+         */
+        getTaxAmount: function (parent, percentage) {
+            var totalPercentage = 0;
```

Comment:
`totalPercentage` variable is declared twice in this file

Reply:
Hi @sivaschenko I've updated code. Thanks :+1: